project-fortis-interfaces/src/actions/Admin/index.js

```diff
@@ -21,6 +26,52 @@ const _methods = {
     }));
   },
 
+  load_users() {
+    const self = this;
+    AdminServices.fetchUsers((err, response, body) => ResponseHandler(err, response, body, (error, graphqlResponse) => {
+      if (graphqlResponse && !error) {
+        let users = graphqlResponse.users.users;
+        users = addIdsToUsersForGrid(users);
+        self.dispatch(constants.ADMIN.LOAD_USERS, { response: users });
+      } else {
+        const action = 'failed';
+        console.error(`[${error}] occured while loading users.`);
+        self.dispatch(constants.ADMIN.LOAD_FAIL, {action});
+      }
+    }));
+  },
+
+  add_users(users) {
+    AdminServices.addUsers(users, (err, response, body) => ResponseHandler(err, response, body, (error, graphqlResponse) => {
+      if (graphqlResponse && !error) {
+        const usersBeforeSave = this.flux.stores.AdminStore.dataStore.users.filter(user => user.id === `${user.identifier}-${user.role}`);
+        const usersAdded = addIdsToUsersForGrid(graphqlResponse.addUsers.users);
+        const usersAfterSave = usersBeforeSave.concat(usersAdded).filter(user => user.identifier !== "" || user.role !== "");
+        this.dispatch(constants.ADMIN.LOAD_USERS, {action: 'saved', response: usersAfterSave});
+      } else {
+        console.error(`[${error}] occured while processing user save request.`);
+        this.dispatch(constants.ADMIN.LOAD_FAIL, {action: 'failed'});
+      }
+    }));
+  },
+
+  remove_users(users, callback) {
+    AdminServices.removeUsers(users, (err, response, body) => ResponseHandler(err, response, body, (error, graphqlResponse) => {
+      if (graphqlResponse && !error) {
+        const usersBeforeRemove = this.flux.stores.AdminStore.dataStore.users;
+        const usersRemoved = addIdsToUsersForGrid(graphqlResponse.removeUsers.users);
+        let usersAfterRemove = getListAfterRemove(usersBeforeRemove, usersRemoved, 'id')
+        usersAfterRemove = usersAfterRemove.filter(user => user.identifier !== "" || user.role !== "");;
+        this.dispatch(constants.ADMIN.LOAD_USERS, {action: 'saved', response: usersAfterRemove});
+        if (callback) callback(null, usersAfterRemove);
```

Comment:
You can avoid the conditional by using the [null object pattern](https://sourcemaking.com/design_patterns/null_object) and defaulting the callback to a [doNothing](https://github.com/CatalystCode/project-fortis/blob/e52cb9f2b656cb162d93d4ffb11a5313a2cb656f/project-fortis-interfaces/src/utils/Utils.js#L108-L109) function at the start of the method. The advantage of this is that you don't have to check if callback is null at every call-site.

```js
// at start of method: set default argument values
callback = callback != null ? callback : doNothing;
// ...
// later in code: can just use callback without caring if it's set or not
callback(null, foo);
```